src/mount/mount-tool.c

```diff
@@ -1104,27 +1104,40 @@ static int acquire_removable(struct udev_device *d) {
         return 1;
 }
 
+static int discover_loop_backing_file(void) {
+        _cleanup_free_ char *escaped = NULL;
+
+        if (arg_mount_where)
+                return 0;
+
+        escaped = xescape(basename(arg_mount_what), "\\");
+        if (!filename_is_valid(escaped))
+                return 0;
+
+        arg_mount_where = strjoin("/run/media/system/", escaped);
```

Comment:
hmm, I think we should search for the file by iterating through all loopback devices with udev, always querying the `loop/backing_file` attribute, which tells us the file name, which we can then compare. (i.e. see `/sys/class/block/loop0/loop/backing_file`). That way things will work for any kind of loopback mount, created by anyone, not just by us....